<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>528</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Mesh_Computational_Geometry</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="Guidebar">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="original_mode">
          <property name="text">
           <string>original plot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="connex_mode">
          <property name="text">
           <string>connexion triangle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="cir_mode">
          <property name="text">
           <string>test circulateur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="check_create_mesh">
          <property name="text">
           <string>drawwire2D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="check_loc">
          <property name="text">
           <string>check_loc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="draw_curv">
          <property name="text">
           <string>draw_curv</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="load_button">
          <property name="text">
           <string>Check_create_mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="flip_edge">
          <property name="text">
           <string>flip edge</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="incre_delaunay">
          <property name="text">
           <string>test_add_point</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loc_point">
          <property name="text">
           <string>test_loc_point</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit">
          <property name="text">
           <string>exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="GLDisplayWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLDisplayWidget</class>
   <extends>QWidget</extends>
   <header>gldisplaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
